src/components/atoms/Paragraph/Paragraph.ts

```diff
@@ -3,17 +3,17 @@ import type { ThemeTypes } from '../../../theme/appTheme';
 
 interface ParagraphInterface {
   theme: ThemeTypes;
-  big?: boolean;
+  $big?: boolean;
```

Comment:
In this PR I am only clearing the browsers console from errors. The decision regarding the sizes should be made by UX or a person who would like to take care of this task and standardise the sizes. This is actually one of our biggest problem in this project. Since we have so few people who contribute to frontend, we agreed not to focus on UX at all for now :( And you can find many places like this and lack of consistency.

Reply:
Let me know if you would like to know more in this topic or would like to discuss. Also maybe you have already noticed, but Żeton doesn't really have proper TypeScript types in many places. We have just made the configuration and we didn't have time to do more then that. Maybe that should be our number one prio?